Dockerfile.CompressImages

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS dotnet
+
+COPY . /src/
+RUN mkdir -p /home/site/wwwroot
+RUN cd /src/CompressImagesFunction && dotnet publish CompressImagesFunction.csproj -c Release --output /home/site/wwwroot
+
+# Native Binaries
+RUN cd /src/CompressImagesFunction && cp bin/Release/netstandard2.0/bin/runtimes/linux-x64/native/libgit2-4aecb64.so /home/site/wwwroot/bin/
+RUN cd /src/CompressImagesFunction && cp bin/Release/netstandard2.0/bin/runtimes/linux-x64/native/Magick.NET-Q16-x64.Native.dll.so /home/site/wwwroot/bin/
```

Comment:
Maybe copy `*.so` here?

Reply:
copy every `*.so` file in the bin?
It would make upgrading libgit slightly easier, but it feels kind of reckless